fastpair/rust/src/bluetooth/mod.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+cfg_if::cfg_if! {
+    if #[cfg(windows)] {
+        mod windows_ble;
+        use windows_ble as imp;
+    } else {
+        compile_error!("Unsupported target platform. Currently supported: Windows.");
```

Comment:
may be better to just panic here instead of compile_error just so github actions can pass. would you also mind adding a build rust on windows github action to test the new windows code?